tests/kafkatest/services/monitor/jmx.py

```diff
@@ -41,10 +42,9 @@ def __init__(self, num_nodes, jmx_object_names=None, jmx_attributes=None, jmx_po
         self.jmx_tool_log = os.path.join(root, "jmx_tool.log")
         self.jmx_tool_err_log = os.path.join(root, "jmx_tool.err.log")
 
-    def clean_node(self, node):
+    def clean_node(self, idx, node):
```

Comment:
maybe instead we can do this?

```
def clean_node(self, node, idx=None):
```

then if `idx` is specified we know not to call `self.idx(node)`?